source/_integrations/amcrest.markdown

```diff
@@ -205,6 +206,129 @@ Service data attribute | Optional | Description
 -|-|-
 `entity_id` | no | Name(s) of entities, e.g., `camera.living_room_camera`.
 
+#### Service `ptz_control`
+
+If your Amcrest or Dahua camera supports PT/PTZ, you will be able to pan, tilt or zoom your camera. Zoom is only supported on PTZ
+cameras with optical zoom. It does not support zooming of Varifocal lenses.
```

Comment:
> Zoom is only supported on PTZ cameras with optical zoom. It does not support zooming of Varifocal lenses.

I'm not sure this is true. I tested this on a PTZ camera that I have that only has digital zoom and it works fine. I would recommend removing these sentences.

Reply:
You're right about zoom working fine for a PTZ camera with digital zoom only (not just optical).   In my earlier testing it didn't work from my PTZ camera because of the original error with zoom-in/out reversed!  

However, we should still advise this zoom does not work for non-PTZ cameras with VariFocal lenses.   Often a VariFocal is a "set and forget" for the desired image.   I have a couple of them and rarely  use the zoom after setup, but if we were to add this capability, it would require an update to python-amcrest.

http://<ip>/cgi-bin/devVideoInput.cgi?action= adjustFocus&focus=<focus>&zoom=<zoom>
   focus: float, the range is between 0 and 1; -1 means reset to position 0.
   zoom: float, the range is between 0 and 1; -1 means reset to position 0.

I have no idea how to determine what the Focus values should be, but I have deferred to AutoFocus after any attempt to zoom.   I have tested this manually passing these http commands, and it works fine.  I doubt this is worth the effort to include.  NVR support is the big ticket item that has seen the most requests.

Reply:
> However, we should still advise this zoom does not work for non-PTZ cameras with VariFocal lenses.

Um, it's a PTZ command, so why would you have to warn it doesn't work with non-PTZ cameras? It doesn't work with cows either. Should we warn about that? :stuck_out_tongue_winking_eye: But seriously, I don't really care either way.

Reply:
I think it would be reasonable for someone to "try it and see" if they have a camera capable of optical  **zooming** but not pan and tilt.   I know I would!   Just saving some knuckleheads like me some effort is all.